unit_tests.sh

```diff
@@ -93,33 +93,29 @@ for group in $test_groups ; do
 
 			run_test 1 $0:$LINENO "unknown TEST" TEST=invalid TEST_PKG=valid
 
-			run_test 1 $0:$LINENO "empty catkin workspace" TEST_PKG=valid 'BEFORE_SCRIPT="rm valid"'
+			# TODO(mlautman): Decide if we should keep this test as an empty ros workspace doesn't
```

Comment:
I would also say it can probably be deleted. But sometimes tests are there for a reason

Reply:
This comment was left as a question for @rhaschke. I commented it out since empty ros workspaces don't seem like a compelling reason to fail. Restored